pkg/occlient/occlient.go

```diff
@@ -876,6 +910,81 @@ func (c *Client) UpdateBuildConfig(buildConfigName string, projectName string, g
 	return nil
 }
 
+// AddSupervisorToDC adds the supervisor to the deployment config
+// dcName is the name of the deployment config to be updated
+// builderImage is the name of the builderImage type
+// projectName is the name of the project
+// annotations are the updated annotations for the new deployment config
+func (c *Client) AddSupervisorToDC(dcName string, builderImage string, projectName string, annotations map[string]string, labels map[string]string) error {
+	appRootVolumeName := fmt.Sprintf("%s-s2idata", dcName)
+
+	dc, err := c.GetDeploymentConfigFromName(dcName)
+	if err != nil {
+		return errors.Wrapf(err, "unable to get DeploymentConfig %s", dcName)
+	}
+
+	dc.Annotations = annotations
+
+	c.AddBootstrapInitContainer(dc, dcName)
+
+	c.AddBootstrapVolume(dc, dcName)
+
+	c.AddBootstrapVolumeMount(dc, dcName)
+
+	dc, err = c.appsClient.DeploymentConfigs(c.namespace).Update(dc)
+	if err != nil {
+		return errors.Wrapf(err, "unable to uDeploymentConfig config %s", dcName)
+	}
+	_, err = c.CreatePVC(appRootVolumeName, "1Gi", labels)
+	if err != nil {
+		return errors.Wrapf(err, "unable to create PVC for %s", dcName)
+	}
+	return nil
+}
+
+// RemoveSupervisorFromDC removes the supervisor from the deployment config
+// dcName is the name of the deployment config to be updated
+// builderImage is the name of the builderImage type
+// projectName is the name of the project
+// annotations are the updated annotations for the new deployment config
+func (c *Client) RemoveSupervisorFromDC(dcName string, builderImage string, projectName string, annotations map[string]string) error {
```

Comment:
similar concern as with `AddSupervisorToDC`

Reply:
@kadel Can you suggest any name? I am confused here too